DigitalLearningSolutions.Web/Views/Register/Password.cshtml

```diff
@@ -0,0 +1,61 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Register
+@model PasswordViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = errorHasOccurred ? "Error: Register - Password": "Register - Password";
+  var passwordError = ViewData.ModelState["Password"]?.Errors?.Count > 0;
+  var confirmPasswordError = ViewData.ModelState["ConfirmPassword"]?.Errors?.Count > 0;
+  var passwordInputErrorClass = passwordError ? "nhsuk-select--error" : "";
+  var passwordFormErrorClass = passwordError ? "nhsuk-form-group--error" : "";
+  var confirmPasswordInputErrorClass = confirmPasswordError ? "nhsuk-select--error" : "";
+  var confirmPasswordFormErrorClass = confirmPasswordError ? "nhsuk-form-group--error" : "";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <form method="post" asp-action="Password">

```

Comment:
Just had a thought - it's probably worth having anti-forgery tokens in these forms if you haven't already added them: https://docs.microsoft.com/en-us/aspnet/web-api/overview/security/preventing-cross-site-request-forgery-csrf-attacks

Otherwise someone who knows the POST url could get users to change their passwords involuntarily since we're using cookie auth.

Reply:
We already do I'm pretty sure, visiting the website sets a `.AspNetCore.Antiforgery` cookie

Reply:
[This is done automatically](https://docs.microsoft.com/en-us/aspnet/core/security/anti-request-forgery?view=aspnetcore-3.1#aspnet-core-antiforgery-configuration), thanks @DanBloxham-sw for the link 😀